index.js

```diff
@@ -98,7 +114,7 @@ Buffer.from
     self.incomingClaimSubmitted = ctx.backend.getMaxValueTracker('incoming_claim_submitted')
 
     ctx.rpc.addMethod('ripple_channel_id', () => {
-      return self.outgoingPaymentChannelId || null     
+      return self.outgoingPaymentChannelId || null
```

Comment:
if this method took one parameter `incomingPaymentChannelId`, and set that parameter on self, it would speed up the connection. When pulling your peer's `ripple_channel_id` you'd be simultaneously pushing your channel ID to them

Reply:
Not sure. If we would do that a peer could overwrite the `incomingPaymentChannelId`, which might cause trouble if the old payment channel has already claims against it.